src/plugins/subsystems/drm/drm.sh

```diff
@@ -208,22 +216,41 @@ function gui_control()
   local isolate_target
   local remote
   local port
+  local set_default=false
```

Comment:
Use single-quotes as in `set_default='false'`

Reply:
:+1: